.github/workflows/scripts/test_disable_statements_check.py

```
@@ -0,0 +1,268 @@
+
+    def test_multiple_violations_same_file(self) -> None:
+        """Test multiple violations in the same file."""
+        content = """
+        // eslint-disable no-console
+        /* istanbul ignore next */
+        it.skip('test', () => {});
+        """
+        violations = self.check_file_content_for_testing(content, 'test.js')
+        self.assertEqual(len(violations), 3)
+
+    def check_file_content_for_testing(self, content: str, file_path: str) -> list:
```

Ruff (0.14.8)

Unused method argument: `file_path`

(ARG002)